ocd/src/memory/romtable/mod.rs

```diff
@@ -147,6 +153,9 @@ impl RomTable {
                 .filter_map(Result::ok)
                 .filter_map(|raw_entry| {
                     let entry_base_addr = raw_entry.component_addr();
+                    if !raw_entry.entry_present {
+                        return None
+                    }
```

Comment:
This is the important bit of the commit. The other one should be a no-op.

Reply:
No, that does not work.

Reply:
This is basically a lookahead. Without it we'll start processing the next RomTable element which in case of the STM32L053 is identified as a RomTable so we're recursing into it which is bad(tm).

Reply:
Sorry, I was referring to the other part of the commit (I guess I shouldn't have put the comment here)

Reply:
Ah, yes, I agree. Actually we're already implicitly fusing that.

Reply:
Fuse wont help as it has to be called externally but it has to be invalidated internally.